package.json

```diff
@@ -3,16 +3,19 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "node ./app.js"
+    "start": "nodemon ./app.js"
   },
```

Comment:
This is not recommended.  If you enabled nodemon, you don't see all error logs.   I prefer nodemon manually.  Another reason is when you launch to Heroku, it takes this "start" script.   We don't want to start with nodemon at Heroku.

Reply:
OK thank you I got it